js/libs/keycloak-admin-client/src/utils/fetchWithError.ts

```diff
@@ -19,7 +19,7 @@ export async function fetchWithError(
 
   if (!response.ok) {
     const responseData = await parseResponse(response);
-    throw new NetworkError("Network response was not OK.", {
+    throw new NetworkError(responseData, {
```

Comment:
I don't think putting the response data in as the error message is what we want to do here, if you want to extract the error message you can do something similar to the [PR here](https://github.com/keycloak/keycloak/pull/30710/files#diff-ded85e61d85857ef5bdb41179908fe1fc6115d4e018c1193b42714d27c49979bR29-R38).

Reply:
@jonkoops updated PR to extract error message as done in [PR](https://github.com/keycloak/keycloak/pull/30710/files#diff-ded85e61d85857ef5bdb41179908fe1fc6115d4e018c1193b42714d27c49979bR29-R38)